commands/blue_green_deploy_command.go

```diff
@@ -57,19 +57,17 @@ func (c *BlueGreenDeployCommand) GetPluginCommand() plugin.Command {
 }
 
 func blueGreenDeployCommandFlagsDefiner() CommandFlagsDefiner {
-	return func(flags *flag.FlagSet) map[string]interface{} {
-		optionValues := deployCommandFlagsDefiner()(flags)
-		delete(optionValues, skipTestingPhase)
```

Comment:
what about `skipTestingPhase`?

Reply:
The flag can't be undefined but it also wasn't an issue because we were accessing the flags' values from a map. This was added to prevent access to this option because it was added with the deployment strategy and isn't part of the `bg-deploy` command. But this option value isn't accessed anyway, so it's safe to remove.